src/app/project_page/project_detail.tsx

```diff
@@ -14,13 +16,15 @@ const ProjectDetail: React.FunctionComponent<{}> = () => {
     const [pod, setPod] = React.useState("")
     const [cpusage, setCpusage] = React.useState("")
 
-    const { projectId } = useParams();
+
+    let { projectId } = useParams();
     //console.log(projectId)
     React.useEffect(() => {
-        axios.get("https://7303ef19-853c-4092-a338-05502ff1eecd.mock.pstmn.io/projectlist/" + projectId).then(res => {
+        axios.get("https://6e905fc6-3dba-44d9-b37b-91bedb726dcf.mock.pstmn.io/projectlist/" + projectId).then(res => {
```

Comment:
@skanthed Sync your PR by pulling in the changes from the remote master

Reply:
The PR #56 will resolve this.